Assets/LeapMotion/Modules/InteractionEngine/Scripts/UI/InteractionToggle.cs

```diff
@@ -72,11 +72,43 @@ public class InteractionToggle : InteractionButton {
     public Action OnUntoggle = () => { };
 
     private float _originalRestingHeight;
+    public float untoggledRestingHeight {
+      get {
+        return _originalRestingHeight;
+      }
+    }
 
-    protected override void Start() {
-      base.Start();
+    /// <summary>
+    /// Returns the local position of this toggle when it is able to relax into its untoggled position.
+    /// </summary>
+    public Vector3 RelaxedToggledLocalPosition {
+      get {
+        return initialLocalPosition + Vector3.back * Mathf.Lerp(minMaxHeight.x, minMaxHeight.y, toggledRestingHeight);
+      }
+    }
+
+    /// <summary>
+    /// Returns the local position of this toggle when it is able to relax into its untoggled position.
+    /// </summary>
+    public override Vector3 RelaxedLocalPosition {
```

Comment:
This is a compile error; you need to also pull in the InteractionButton, which includes the original RelaxedLocalPosition implementation

Reply:
blarg whoops.